infrastructures/kubernetes/src/main/java/org/eclipse/che/workspace/infrastructure/kubernetes/provision/KubernetesTrustedCAProvisioner.java

```diff
@@ -113,26 +112,21 @@ public void provision(KubernetesEnvironment k8sEnv, RuntimeIdentity runtimeID)
     }
 
     KubernetesNamespace namespace = namespaceFactory.getOrCreate(runtimeID);
-    Optional<ConfigMap> existing = namespace.configMaps().get(configMapName);
```

Comment:
why this check is not needed anymore?

Reply:
We do have exactly one configmap per workspace that is created just before the workspace creating. We are not trying to update the content of existed configmap and use it for all workspaces.